nullaway/src/main/java/com/uber/nullaway/handlers/AbstractFieldContractHandler.java

```diff
@@ -60,27 +60,26 @@ protected AbstractFieldContractHandler(String annotName) {
   /**
    * Verifies that the method being processed adheres to the annotation specifications.
    *
-   * @param analysis NullAway instance.
    * @param tree Method tree under processing.
-   * @param state Error Prone {@link VisitorState}.
-   * @param methodSymbol Processing method symbol.
+   * @param methodAnalysisContext The MethodAnalysisContext object
    */
   @Override
-  public void onMatchMethod(
-      NullAway analysis, MethodTree tree, VisitorState state, Symbol.MethodSymbol methodSymbol) {
+  public void onMatchMethod(MethodTree tree, MethodAnalysisContext methodAnalysisContext) {
     Set<String> annotationContent =
-        NullabilityUtil.getAnnotationValueArray(methodSymbol, annotName, false);
+        NullabilityUtil.getAnnotationValueArray(
+            methodAnalysisContext.getMethodSymbol(), annotName, false);
```

Comment:
`getMethodSymbol()` used multiple times in this method and should be stored in a local variable